crates/polars-lazy/src/physical_plan/planner/lp.rs

```diff
@@ -139,11 +139,11 @@ pub fn create_physical_plan(
         PythonScan { options, .. } => Ok(Box::new(executors::PythonScanExec { options })),
         Sink { payload, .. } => match payload {
             SinkType::Memory => {
-                polars_bail!(InvalidOperation: "memory sink not supported in the standard engine")
+                polars_bail!(InvalidOperation: "memory sink not supported in the standard engine.")
             },
             SinkType::File { file_type, .. } => {
                 polars_bail!(InvalidOperation:
-                    "sink_{file_type:?} not yet supported in standard engine. Use 'collect().write_parquet()'"
+                    "sink_{file_type:?} not yet supported in standard engine."
```

Comment:
You can feature gate the error message. I want to keep the proper error message for python.

Reply:
I'm not sure what your intent is, is 6168f1f09b79aa27c48d0625ce15e20aa453e768 okay?